package.json

```diff
@@ -17,6 +17,7 @@
     "add": "^2.0.6",
     "axios": "^0.27.2",
     "browser-image-compression": "^2.0.0",
+    "classnames": "^2.3.2",
```

Comment:
Why do you need it?

Reply:
I'm using `classnames` module in few components to break down the `className` string into multiple lines and organize it better. For example in `src/components/Pitch.js` I've used the module in 2 elements to separate its variables and styles.

If this module's usage is unnecessary, I will remove it.